GitVersion.yml

```
@@ -6,6 +6,12 @@ branches:
     tag: 'alpha'
     increment: Patch
     source-branches: []
+  hotfix:
+    regex: hotfix[/-]
+    source-branches: [develop]
+    prevent-increment:
+      when-current-commit-tagged: true
+    is-release-branch: true 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)